scripts/gWMS-CMSRunAnalysis.sh

```diff
@@ -193,6 +193,9 @@ then
     CRAB_localOutputFiles=`grep '^CRAB_localOutputFiles =' $_CONDOR_JOB_AD | tail -1 | tr -d '"' | tr -d ',' | sed 's/CRAB_localOutputFiles = //'`
     export CRAB_Id=`grep '^CRAB_Id =' $_CONDOR_JOB_AD | tr -d '"' | tail -1 | awk '{print $NF;}'`
     export CRAB_Retry=`grep '^CRAB_Retry =' $_CONDOR_JOB_AD | tail -1 | tr -d '"' | awk '{print $NF;}'`
+    export CRAB_Reqname=`grep '^CRAB_Reqname =' $_CONDOR_JOB_AD | tail -1 | tr -d '"' | awk '{print $NF;}'`
```

Comment:
Have you tested it? because I am not sure that grep is case sensitive, and I think that the "original" classad is `CRAB_ReqName`: 

```suggestion
    export CRAB_Reqname=`grep '^CRAB_ReqName =' $_CONDOR_JOB_AD | tail -1 | tr -d '"' | awk '{print $NF;}'`
```

See from [here](https://cmsweb.cern.ch:8443/scheddmon/059/dmapelli/240423_114530:dmapelli_crab_throttle_20240423_134526/job_out.2.0.txt) 

```plaintext
== JOB AD: CRAB_ReqName = "240423_114530:dmapelli_crab_throttle_20240423_134526"
```

Reply:
oops Thanks !
The fact that classAd are "almost" case-insensitive often gets me confused. No I have not tested